test-types/empty.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * This has to be a empty file
```

Comment:
what's the purpose of this file?

Reply:
It's a TSD limitation and this file is a "temporary" fix for it. I've described the problem here https://github.com/MLH-Fellowship/jest-runner-tsd/#for-typescript-projects

Reply:
aha. add that as a comment?

Reply:
Sure, done that :+1: